src/Indicator.vala

```diff
@@ -86,6 +89,95 @@ public class Network.Indicator : Wingpanel.Indicator {
             popover_widget.closed ();
         }
     }
+
+    private void update_tooltip () {
+        switch (popover_widget.state) {
+            case Network.State.CONNECTING_WIRED:
+                /* If there's only one active ethernet connection,
+                we get back the string "Wired". We won't want to
+                show the user Connecting to "Wired" so we'll have
+                to show them something else if we get back
+                "Wired" from get_active_wired_name () */
+
+                string active_wired_name = get_active_wired_name ();
+
+                if (active_wired_name == "Wired") {
+                    display_widget.tooltip_markup = _("Connecting to wired network");
+                } else {
+                    display_widget.tooltip_markup = _("Connecting to “%s”").printf (get_active_wired_name ());
```

Comment:
You cache the string value here but then don't use it in the tooltip markup. Also, this string is probably translated so in order to make this match for all locales, this should be:
```suggestion
                var active_wired_name = get_active_wired_name ();
                if (active_wired_name == _("Wired")) {
                    display_widget.tooltip_markup = _("Connecting to wired network");
                } else {
                    display_widget.tooltip_markup = _("Connecting to “%s”").printf (active_wired_name);
```

Reply:
Ahhh can't believe I missed that. It's because I copied and pasted the wifi one and switched out the names (with the if statement coming in a later commit so I forgot about re-using `active_wired_name`). Thanks for pointing it out!

Reply:
Also, is there any significance between using string vs var? I'm assuming var just automatically determines the variable type?

Reply:
Yeah, it's just code style. I don't think there's any realistic difference, but in general we tend to prefer implicit typing